src/connectRoutes.js

```diff
@@ -317,6 +329,9 @@ export default (
     }
 
     if (typeof window !== 'undefined') {
+      // `kind` is typed as ?string which will not throw if passed to .test.
+      // Typing kind as string threw other flow errrs in the codebase
+      // $FlowIssue
```

Comment:
Open to suggestions as to how to improve type coverage here. Since kind is `?string` flow does no like passing it to `test`. However going from `?string` -> `string` caused a slew of other flow related issues in the codebase.